### ISSUE TITLE
More user friendly actions - add title for action

### ISSUE THREAD

**Issue by petr-fischer** [derf/feh#224]
hello, will be neat to have possibility to define title/name for action - and if title is defined, show it instead of command after "a" key is pressed

I have action for linking image to favorites (+ dunst notify for feedback), something like:
```
NOTIFY_TIMEOUT=3000
NOTIFY="notify-send.sh -u low -a feh -t $NOTIFY_TIMEOUT"
feh .... --action="ln -s '`pwd`/%f' '/home/pf/Photos/links/favorites/`date +%Y.%m.%d-%H.%M.%S`-%n'; $NOTIFY 'Added to favorites:\n`pwd`/%f'"
```

a few similar commands and action menu look like a mess (when showed with "a" key)

so why not something like this:
```
feh ... --action="[title for action] ln -s.... etc etc"
```

your opinion?

thanks!

**Comment by derf** [comment]
Good point indeed. I'll have to think a bit about the implementation details (making --action more special versus adding yet more arguments — Probably the former as you bsuggested), but this might just be one of the few feature request I actually implement in a timely manner :)

**Comment by derf** [comment]
Included in feh 2.15
